torch/csrc/distributed/autograd/context/dist_autograd_container.cpp

```diff
@@ -23,8 +24,17 @@ DistAutogradContainer::DistAutogradContainer()
       worker_id_(0),
       initialized_(false),
       next_autograd_message_id_(0),
-      max_id_(0) {}
+      max_id_(0) {
+  cleanupContextTimeout = std::chrono::seconds(600);
```

Comment:
nit: `cleanupContextTimeout_` and try to initialize all members in the constructor initializer list.